lib/server-impl.js

```diff
@@ -26,7 +26,7 @@ function createApp(options) {
     );
 
     const app = getApp(config);
-    const server = app.listen(app.get('port'), () => {
+    const server = app.listen(app.get('port'), options.host, () => {
```

Comment:
We can use an object instead of multiple arguments: https://nodejs.org/api/net.html#net_server_listen_options_callback